package.json

```diff
@@ -77,6 +77,7 @@
     "@mui/internal-scripts": "^2.0.7",
     "@mui/internal-test-utils": "^2.0.7",
     "@mui/monorepo": "github:mui/material-ui#v7.0.2",
+    "@mui-public/babel-plugin-display-name": "github:romgrk/mui-public#feat-babel-plugin-display-name&path:./packages/babel-plugin-display-name",
```

Comment:
Following the discussion in https://mui-org.slack.com/archives/C042YB5RB3N/p1747198755981039

NOTE: Replace `romgrk/mui-public` before merging.